src/molgenis/capice/cli/args_handler_predict.py

```diff
@@ -7,6 +7,8 @@
 from molgenis.capice.main_predict import CapicePredict
 from molgenis.capice.core.capice_manager import CapiceManager
 from molgenis.capice.cli.args_handler_parent import ArgsHandlerParent
+from re import match
```

Comment:
Please sort import statements as following:

- python native libaries
- [whiteline]
- project imports

Also please sort them by length of the import statement